content/ngrx/do-not-put-everything-in-the-store.md

```diff
@@ -6,6 +6,9 @@ title: don't put everything in the store
 
 `@ngrx/store` (or Redux in general) provides us with a lot of great features and can be used in a lot of use cases. But sometimes this pattern can be an overkill. Implementing it means we get the downside of using Redux (a lot of extra code and complexity) without benefiting of the upsides (predictable state container and unidirectional data flow).
 
+Don't put everything in a store. Persist information in a database, use routes to reflect applied filters on a list or use a Behavior Subject in a service if you need to save some simple data like settings.
```

Comment:
Like the addition. It might be better to add this with the solution part though ;-). Could you move it there?

Reply:
Good idea @KwintenP. I moved it to the solutions part, improved the wording a little bit and added a link to Juri's great article. I also squashed the commits for a clean history.